pandas/tests/indexes/conftest.py

```diff
@@ -47,3 +47,37 @@ def zero(request):
     # For testing division by (or of) zero for Index with length 5, this
     # gives several scalar-zeros and length-5 vector-zeros
     return request.param
+
+
+def _get_subclasses(cls):
+    for subclass in cls.__subclasses__():
+        yield from _get_subclasses(subclass)
+        yield subclass
+
+
+all_indexes = [index for index in ([Index] + list(set(_get_subclasses(Index))))
+               if getattr(pd, index.__name__, None) is not None]
```

Comment:
What is the name check for?

Reply:
Can just simplify to `if getattr(pd, index.__name__, False)`

Reply:
missed that. thanks.